sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -92,6 +88,9 @@
 POWER_UNIT = 'dBm'
 BIAS_UNIT = 'mA'
 
+PLATFORM_PMON_CTRL_FILENAME = "pmon_daemon_control.json"
+ENABLE_SFF_MGR_FLAG_NAME = "enable_xcvrd_sff_mgr"
+
```

Comment:
@longhuan-cisco not required if we use cmis mgr skip approach

Reply:
raised another PR to use the old approach:  https://github.com/sonic-net/sonic-buildimage/pull/18054